language/llama2-70b/user.conf

```diff
@@ -9,3 +9,5 @@
 *.Server.target_qps = 0.5
 *.Server.min_duration = 120000
 *.Server.min_query_count = 100
+
+llama2-70b.Server.sample_concatenate_permutation = 1
```

Comment:
Can you add gptj.Server as well (because of the same reason)?

Reply:
I added it in the `mlperf.conf` file. Just in case in the future it applies